src/Actions.Core/build/GitHub.Actions.Core.props

```diff
@@ -0,0 +1,16 @@
+<Project>
+    <PropertyGroup>
+        <!-- Provide a global opt-out for our modifications - a user can set IsTargetingGithubActions to false to make sure we don't do anything -->
+        <IsTargetingGithubActions 
+            Condition="'$(IsTargetingGithubActions)' == ''" >true</IsTargetingGithubActions>
```

Comment:
safety net: we can document this in the README and tell users to set this to false if they don't want these defaults.